ShopHub/ShopHub/ContentView.swift

```diff
@@ -15,18 +15,16 @@ struct ContentView: View {
     var body: some View {
         TabView {
             MenuTabView()
-                .onAppear {
-                    DispatchQueue
-                        .main
-                        .asyncAfter(deadline: .now() + 3) {
-                            launchScreenManager.dismiss()
-                        }
-                }
         }
+        .onAppear(perform: dimissLaunchScreen)
```

Comment:
Wow, I didn't realize these parameters can do the same jobs. Now, I learned them, Thanks!